dataprofiler/profilers/unstructured_text_profile.py

```diff
@@ -0,0 +1,223 @@
+            'having', 'do', 'does', 'did', 'doing', 'a', 'an', 'the', 'and',
+            'but', 'if', 'or', 'because', 'as', 'until', 'while', 'of', 'at',
+            'by', 'for', 'with', 'about', 'against', 'between', 'into',
+            'through', 'during', 'before', 'after', 'above', 'below', 'to',
+            'from', 'up', 'down', 'in', 'out', 'on', 'off', 'over', 'under',
+            'again', 'further', 'then', 'once', 'here', 'there', 'when',
+            'where', 'why', 'how', 'all', 'any', 'both', 'each', 'few', 'more',
+            'most', 'other', 'some', 'such', 'no', 'nor', 'not', 'only', 'own',
+            'same', 'so', 'than', 'too', 'very', 's', 't', 'can', 'will',
+            'just', 'don', "don't", 'should', "should've", 'now', 'd', 'll',
+            'm', 'o', 're', 've', 'y', 'ain', 'aren', "aren't", 'couldn',
+            "couldn't", 'didn', "didn't", 'doesn', "doesn't", 'hadn', "hadn't",
+            'hasn', "hasn't", 'haven', "haven't", 'isn', "isn't", 'ma',
+            'mightn', "mightn't", 'mustn', "mustn't", 'needn', "needn't",
+            'shan', "shan't", 'shouldn', "shouldn't", 'wasn', "wasn't",
+            'weren', "weren't", 'won', "won't", 'wouldn', "wouldn't"
+        }
+
+        self.__calculations = {
+            "vocab": TextProfiler._update_vocab,
+            "words": TextProfiler._update_words,
+        }
+        BaseColumnProfiler._filter_properties_w_options(self.__calculations, options)
+
+    def _add_helper(self, other1, other2):
+        """
+        Merges the properties of two BaseColumnProfile objects
+
+        :param other1: first profile
+        :param other2: second profile
+        :type other1: BaseColumnProfiler
+        :type other2: BaseColumnProfiler
+        """
+        if other1.name == other2.name:
+            self.name = other1.name
+        else:
+            raise ValueError("Text names unmatched: {} != {}"
+                             .format(other1.name, other2.name))
+
+        self.times = defaultdict(
+            float, {key: (other1.times.get(key, 0)
+                          + other2.times.get(key, 0)
+                          + self.times.get(key, 0))
+                    for key in (set(other1.times) | set(other2.times)
+                                | set(self.times))}
+        )
+
+        self.sample_size = other1.sample_size + other2.sample_size
+
+    def __add__(self, other):
+        """
+        Merges the properties of two TextProfiler profiles
+
+        :param self: first profile
+        :param other: second profile
+        :type self: TextProfiler
+        :type other: TextProfiler
+        :return: New TextProfiler merged profile
+        """
+        if not isinstance(other, TextProfiler):
+            raise TypeError("Unsupported operand type(s) for +: "
+                            "'TextProfiler' and '{}'".format(
+                other.__class__.__name__))
+        merged_profile = TextProfiler(None)
```

Comment:
we will have a TODO here for sending in the options potentially depending on what might need to be set on instantiation